### PR TITLE
Fail job run if project is failed

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
This is needed b/c of offline syncs now

### CODE REVIEW

File: awx/main/tasks.py
Comment (ryanpetrello) [inline awx/main/tasks.py]: Maybe:

> The Project associated with this Job Template failed to sync.

If you don't know Tower API terminology, this sentence may not be super clear.

Comment (AlanCoding) [reply awx/main/tasks.py]: I don't like the use of "sync" there, because in this case we avoid running the sync.

You're right, this is user-facing so what about

> Playbook can not be reliably obtained because its associated project is {status}. Please correct and update the project.

Everyone knows what a playbook is!

Comment (ryanpetrello) [reply awx/main/tasks.py]: > The Project associated with this Job Template failed to update.

😄 ?

I don't love mentioning "project status" if you're not familiar w/ the API's notion of objects having a "status".  UI-only people likely only understand this concept as a red/green dot, not "status".

I wouldn't normally be so picky, but I think it's pretty important to get this message right, since it's likely one of the first errors a user might encounter on install cc @tvo318

**Review by ryanpetrello — APPROVED** [review]
I'm good w/ this as a solution to https://github.com/ansible/tower/issues/3120

File: awx/main/tasks.py
Comment (AlanCoding) [reply awx/main/tasks.py]: > The Project associated with this Job Template failed to update.

I agree that this is very user-friendly, but my problem here is that it's just not true.

what about:

> The Project associated with this Job Template had a failed update.

Comment (wenottingham) [reply awx/main/tasks.py]: "The project revision for this job template is unknown due to a failed update". ?

**Comment by AlanCoding** [comment]
okay, WIP label off because:

 - testing work has completed
 - agreed upon wording inserted
 - commits squashed

Zuul is working again, so waiting on those results before merge.

**Pull request merged**
